### ISSUE TITLE
Android: custom marker views (non-image) random positions and not updating

### ISSUE THREAD

**Issue by allthetime** [react-community/react-native-maps#328]
Hi, I've been noticing some strange behaviour with markers in Android.

I was using images before, and sometimes they would load in the wrong position (usually up about 40 pixels), and sometimes they don't. Sometimes the images load a little bigger, and sometimes they load scaled down. 

Anyway, I decided I wanted to use a view instead, but it still suffers from the positioning issue. 

Also, if I try and dynamically update the view (like to highlight a specific marker), the changes are not reflected on the map and the marker stays in its original state. In contrast, this works just fine on iOS.

Looking through the issues it seems the positioning problem has been around for a while and was supposedly fixed, but this is not the case. Does anyone know what's going on, or how it could be fixed/patched?

**Comment by nillo** [comment]
Hi there i'am having the same issue, but issue started from react 0.28+  , we are currently running our project on 0.27.2 , and the only reason we did not upgrade to 29 is because, of this issue.
'we checked the native code and the javascript code, but there aren't much differences within the react-native-maps code. so no clue on how to fix this.

i could try to test with this version, actually u can update tot 0.6.0 it won't do harm
   "react-native-maps": "^0.5.0",
    "react": "^15.1.0",
    "react-native": "^0.27.2", 

try it in this combination if possible, but i hope to see some fix or something soon
